securedrop/source_templates/utils.html

```diff
@@ -13,7 +13,7 @@ <h2 id="codename-heading" hidden>{{ gettext('Codename') }}</h2>
 {% endif %}
 
   <input id="codename-show-checkbox" type="checkbox" role="button"{% if not new %} checked {% endif -%}
-    aria-label="{{ gettext('Show Codename') }}" aria-controls="codename">
+    aria-label="{{ gettext('Make Codename Visible on the Display') }}" aria-controls="codename">
```

Comment:
Hmm: it might not _meaningfully_ be, though, for a user who's navigating exclusively by screen-reader.  What about:

```suggestion
    aria-label="{{ gettext('Reveal Codename') }}" aria-controls="codename">
```

In fact, I wonder if "reveal" is the ability- and interface-agnostic term for what this widget does here.  Just thinking out loud.  :-)

Reply:
That's a great distinction, and I think your instinct of using "reveal" is ideal here.